<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(65, 65, 48);
</string>
  </property>
  <widget class="QLabel" name="label_codigoEDITAR">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>CÓDIGO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fornecedorEDITAR">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>FORNECEDOR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_precoEDITAR">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>PREÇO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_detalhesEDITAR">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>DETALHES</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nomeEDITAR">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>NOME</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>16</width>
     <height>201</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_codED">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid #ffffff;
color: rgb(254, 255, 153);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_fornED">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid #FFFFFF;
color: rgb(254, 255, 153);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_nomeED">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid #FFFFFF;
color: rgb(254, 255, 153);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_detED">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid #FFFFFF;
color: rgb(254, 255, 153);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_precED">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid #FFFFFF;
color: rgb(254, 255, 153);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_codNOVO">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(254, 255, 153);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fornNOVO">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(254, 255, 153);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nomeNOVO">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>200</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(254, 255, 153);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_detNOVO">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>240</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(254, 255, 153);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_precNOVO">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>280</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(254, 255, 153);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_codigoEDITAR_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>120</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>CÓDIGO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fornecedorEDITAR_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>FORNECEDOR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nomeEDITAR_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>200</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>NOME</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_detalhesEDITAR_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>240</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>DETALHES</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_precoEDITAR_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>PREÇO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Digite o código do Produto</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pesqEDITAR">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pesqEDITAR">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Pesquisar</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>383</x>
     <y>110</y>
     <width>20</width>
     <height>201</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>NOVO DADOS DO PRODUTO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>PRODUTO PESQUISADO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_editprodNOVO">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
border: 2px solid #111111;
border-radius: 8px;
background-color: rgb(254, 255, 188);</string>
   </property>
   <property name="text">
    <string>Editar Produto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>340</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
border: 2px solid #111111;
border-radius: 8px;
background-color: rgb(254, 255, 188);</string>
   </property>
   <property name="text">
    <string>limpar</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>222</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_editprodNOVO_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
border: 2px solid #111111;
border-radius: 8px;
background-color: rgb(254, 255, 188);</string>
   </property>
   <property name="text">
    <string>Novo Produto</string>
   </property>
  </widget>
  <zorder>label_codigoEDITAR</zorder>
  <zorder>label_fornecedorEDITAR</zorder>
  <zorder>label_precoEDITAR</zorder>
  <zorder>label_detalhesEDITAR</zorder>
  <zorder>label_nomeEDITAR</zorder>
  <zorder>line</zorder>
  <zorder>label_codED</zorder>
  <zorder>label_fornED</zorder>
  <zorder>label_nomeED</zorder>
  <zorder>label_precED</zorder>
  <zorder>lineEdit_codNOVO</zorder>
  <zorder>lineEdit_fornNOVO</zorder>
  <zorder>lineEdit_nomeNOVO</zorder>
  <zorder>lineEdit_detNOVO</zorder>
  <zorder>lineEdit_precNOVO</zorder>
  <zorder>label_codigoEDITAR_2</zorder>
  <zorder>label_fornecedorEDITAR_2</zorder>
  <zorder>label_nomeEDITAR_2</zorder>
  <zorder>label_detalhesEDITAR_2</zorder>
  <zorder>label_precoEDITAR_2</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit_pesqEDITAR</zorder>
  <zorder>pushButton_pesqEDITAR</zorder>
  <zorder>line_2</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButton_editprodNOVO</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>line_3</zorder>
  <zorder>line_4</zorder>
  <zorder>line_5</zorder>
  <zorder>line_6</zorder>
  <zorder>line_7</zorder>
  <zorder>label_detED</zorder>
  <zorder>pushButton_editprodNOVO_2</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_precNOVO</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_detNOVO</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>641</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_nomeNOVO</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>651</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_fornNOVO</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>665</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_codNOVO</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>691</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>628</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_editprodNOVO_2</sender>
   <signal>clicked()</signal>
   <receiver>label_precED</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_editprodNOVO_2</sender>
   <signal>clicked()</signal>
   <receiver>label_detED</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_editprodNOVO_2</sender>
   <signal>clicked()</signal>
   <receiver>label_nomeED</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_editprodNOVO_2</sender>
   <signal>clicked()</signal>
   <receiver>label_fornED</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_editprodNOVO_2</sender>
   <signal>clicked()</signal>
   <receiver>label_codED</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_editprodNOVO_2</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_pesqEDITAR</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
